pkg/kubelet/stats/cri_stats_provider_windows.go

```diff
@@ -20,104 +20,101 @@ limitations under the License.
 package stats
 
 import (
-	"fmt"
 	"time"
 
-	"github.com/Microsoft/hcsshim"
+	"k8s.io/klog/v2"
 
+	"github.com/Microsoft/hcsshim"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/klog/v2"
 	statsapi "k8s.io/kubelet/pkg/apis/stats/v1alpha1"
 )
 
+// windowsNetworkStatsProvider creates an interface that allows for testing the logic without needing to create a container
+type windowsNetworkStatsProvider interface {
+	HNSListEndpointRequest() ([]hcsshim.HNSEndpoint, error)
+	GetHNSEndpointStats(endpointName string) (*hcsshim.HNSEndpointStats, error)
+	Now() time.Time
```

Comment:
Not sure Now() belongs in the interface.  Adding a k8s.io/utils/clock.Clock to the cri stats provider seems better.

Reply:
makes sense, i've made this adjustment